dashboard/test/ui/runner.rb

```diff
@@ -283,12 +283,19 @@ def output_synopsis(output_text)
     end
   end
 
-  if succeeded
-    HipChat.log "<b>dashboard</b> UI tests passed with <b>#{test_run_string}</b> (#{format_duration(test_duration)})"
+  parsed_output = output_stdout.match(/^(?<scenarios>\d+) scenarios?( \((?<info>.*?)\))?/)
+  scenario_count = parsed_output[:scenarios].to_i
+  scenario_info = parsed_output[:info]
+  scenario_info = ", #{scenario_info}" unless scenario_info.blank?
+
+  if scenario_count == 0
```

Comment:
mostly just being paranoid, I do not know of any cases where this would be true but some possibilities: .feature file has bad syntax, we called cucumber with bad options, cucumber crashed before it tried to parse the feature file.